woocommerce/woocommerce-android
# Activate: Reorganize the notifications code

Fixes #9011, which is a subtask of #8999.

This PR doesn't change any functionality, it just refactors the packages for notifications to reduce the noise in subsequent PRs. All the notification code has been moved to the root package `notifications`, with `local` and `push` subfolders.

**To test:**
Just a simple smoke test is fine -- create a new order and make sure a notification is displayed.
